pkg/virt-launcher/virtwrap/access-credentials/access_credentials.go

```diff
@@ -312,6 +313,15 @@ func (l *AccessCredentialManager) agentSetAuthorizedKeys(domName string, user st
 		}
 		defer domain.Free()
 
+		if len(authorizedKeys) == 0 {
+			// The dummy authorized keys is a workaround for a bug in the libvirt-go-module library.
+			// It will panic, if the keys argument is empty.
+			//
+			// TODO: Remove the workaround once a new version of libvirt-go-module is released,
```

Comment:
So is it worth waiting?

Reply:
We can wait. I'm not sure when the new release will be, but looking at previous release times, it may not be more than a month away.
We will still need this PR, but just without the dummy data.